app/pages/BuyNBTC/SellNBTCTabContent.tsx

```diff
@@ -139,10 +136,13 @@ export function SellNBTCTabContent() {
 					</span>
 				</div>
 				{isSuiWalletConnected ? (
-					<Button type="submit" disabled={isPending} isLoading={isPending} layout="oneLine">
-						Sell nBTC
-						<ChevronRight />
-					</Button>
+					<button type="submit" disabled={isPending} className="btn btn-primary">
+						<div className="flex w-full gap-2 items-center justify-center">
```

Comment:
remove wrapper. If needed we can add more class to the button element.

Reply:
Done